src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEngine.Unix.cs

```diff
@@ -387,5 +445,17 @@ private bool TryRegister(SafeSocketHandle socket, IntPtr handle, out Interop.Err
                 Interop.Sys.SocketEvents.Read | Interop.Sys.SocketEvents.Write, handle);
             return error == Interop.Error.SUCCESS;
         }
+
+        private struct Event
```

Comment:
readonly

Reply:
+1 for readonly, I think that we could also give it a less generic name, `EpollEvent` for example.

```suggestion
        private readonly struct EpollEvent
```

Reply:
BTW do you think that changing `SocketEvent` size from 4 bytes to 1 could improve the perf in any way?

https://github.com/dotnet/runtime/blob/6ef538d46fc971f351a22cbe854717f4ada59d89/src/libraries/Common/src/Interop/Unix/System.Native/Interop.SocketEvent.cs#L13

Reply:
You're suggesting that to help reduce the size of EpollEvent?  It wouldn't change it's size; it would just result in more of the same space being padding.

Reply:
Done